source/03-specifications/02-formats-for-counter-reports.rst

```diff
@@ -60,11 +60,11 @@ Table 3.f (below): COUNTER Report Header Elements
      - Example
 
    * - Report_Name
-     - The name of the report as it appears in :numref:`reports-for-libraries`.
+     - The name of the report as it appears in :numref:`reports`.
      - Journal Requests (Excluding OA_Gold)
 
    * - Report_ID
-     - The unique identifier for the report as it appears in :numref:`reports-for-libraries`.
+     - The unique identifier for the report as it appears in :numref:`reports`.
```

Comment:
Same question here.

Reply:
both incorrect, should refer to reports-for-consumers, edited.